config/jobs/ppc64le-cloud/builds/kubernetes-postsubmit.yaml

```diff
@@ -44,6 +44,47 @@ postsubmits:
                 tar -zcvf kubernetes-server-linux-`go env GOARCH`.tar.gz --directory=_output/release-stage/ kubernetes/server/bin
 
                 curl -v -X POST http://build-bot.prow:8090/build -H 'Content-Type: multipart/form-data' -F file1=@kubernetes-test-linux-`go env GOARCH`.tar.gz -F file2=@kubernetes-client-linux-`go env GOARCH`.tar.gz -F file3=@kubernetes-server-linux-`go env GOARCH`.tar.gz -F source=github.com/kubernetes/kubernetes.git -F commit=$GITCOMMIT -F project=kubernetes/master/golang/master
+    - name: postsubmit-master-golang-kubernetes-unit-test-ppc64le
+      cluster: k8s-ppc64le-cluster
+      decorate: true
+      branches:
+        - master
+      run_if_changed: '^golang/master/'
+      extra_refs:
+        - base_ref: master
+          org: kubernetes
+          repo: kubernetes
+          workdir: true
+      spec:
+        containers:
+          - image: quay.io/powercloud/all-in-one:0.1
+            command:
+              - /bin/bash
+            args:
+              - -c
+              - |
+                set -o errexit
+                set -o nounset
+                set -o pipefail
+                set -o xtrace
+
+                export KUBE_TIMEOUT='--timeout=300s'
+                export KUBE_COVER="n"
+                export KUBE_KEEP_VERBOSE_TEST_OUTPUT=y
```

Comment:
Is this enough to generate the artifact? and will it dump the content to `ARTIFACT` folder?

Reply:
Yes it is.
I see the junit xml file being created in artifacts folder where other existing jobs usually create. Please see the sample run that I pushed to gcs bucket:
https://console.cloud.google.com/storage/browser/ppc64le-kubernetes/logs/postsubmit-master-golang-kubernetes-unit-test-ppc64le/1331107525730242560/artifacts;tab=objects?pageState=(%22StorageObjectListTable%22:(%22f%22:%22%255B%255D%22))&authuser=0&project=ibm-kubernetes&prefix=&forceOnObjectsSortingFiltering=false

Also view the sample run in prow dashboard:
https://prow.k8s-prow-c7f9fbfa2832409e2a3a6666e9dc52a3-0000.us-south.containers.appdomain.cloud/view/gs/ppc64le-kubernetes/logs/postsubmit-master-golang-kubernetes-unit-test-ppc64le/1331107525730242560

Reply:
hack/make-rules/test.sh from kubernetes has the below code when KUBE_JUNIT_REPORT_DIR is not passed:

```
KUBE_JUNIT_REPORT_DIR=${KUBE_JUNIT_REPORT_DIR:-}
# If KUBE_JUNIT_REPORT_DIR is unset, and ARTIFACTS is set, then have them match.
if [[ -z "${KUBE_JUNIT_REPORT_DIR:-}" && -n "${ARTIFACTS:-}" ]]; then
    export KUBE_JUNIT_REPORT_DIR="${ARTIFACTS}"
fi

Reply:
okay